keyboards/delikeeb/vanana/keymaps/default/keymap.c

```diff
@@ -0,0 +1,150 @@
+/* Copyright 2020 noclew
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+#ifdef AUDIO_ENABLE
+  float song_coin[][2]  = SONG(COIN_SOUND);
+#endif
+
+// Defines names for use in layer keycodes and the keymap
+enum layer_names {
+    _BASE,
+    _LOWER,
+    _RAISE,
+    _ADJUST
+};
+
+// Defines the keycodes used by our macros in process_record_user
+enum custom_keycodes {
+    BASE,
+    LOWER,
+    RAISE,
+    RGBF,
+    RGBR,
+    VOLUP,
+    VOLDN
+};
+
+#define RGBF RGB_MODE_FORWARD
+#define RGBR RGB_MODE_REVERSE
+#define VOLUP KC__VOLUP
+#define VOLDN KC__VOLDOWN
```

Comment:
```suggestion
```

Reply:
Can I remove these custom keycodes although they are used in keymaps?

Reply:
I've updated my suggestions to completely replace them with core keycodes.

Reply:
Is it not recommended use aliases? I will change as you suggest, but I'd like to know what are the cons when using aliases. I used aliases because it makes a keymap file look a bit more tidy, like KC_VOLU instead of KC_AUDIO_VOL_UP. Can you let me know why it is not recommended?

Reply:
We already have short aliases for all core keycodes.

Reply:
> We already have short aliases for all core keycodes.

Oh, yes. If you were referring to the QMK's default keycodes AND aliases by core keycodes, I updated all my keymaps to use aliases and removed the custom keycodes. Thanks for pointing it out. I forgot that I could use aliases.

Reply:
All done